executable-spec/src/Cardano/Ledger/Spec/STS/Update/Ideation.hs

```diff
@@ -265,6 +274,26 @@ instance
           then pure $! Reveal sip
           else submissionGen
 
+      voteGen
+        :: ActiveSIPs p
+        -> Participants p
+        -> Core.Slot
+        -> QC.Gen (IdeationPayload p)
+      voteGen actsips partnts currSlot =
+        do
+          voter <- QC.elements $ Set.toList $ dom partnts
+          confidence <- QC.frequency [ (60, pure $ Data.For)
```

Comment:
We might want to comment on the chosen frequencies. Readers might ask why not making all the `frequency` weights equal.

Reply:
Also don't forget to check `hlint` for redundant `$`.

Reply:
Done OK